### ISSUE TITLE
ERROR TypeError: "browserLang is undefined"

### ISSUE THREAD

**Issue by mmrzk** [ngneat/transloco#289]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior
When I remove all the preffered languages for displaying pages in Firefox (Settings => General => Language => press [Choose...] button => Webpage Language Settings box appears => remove all the languages) and call the `getBrowserLang()` function in code it generates the `ERROR TypeError: "browserLang is undefined"` error message in console and crashes the app. 
The error reffers to `ngneat-transloco.js:2081`.

## Expected behavior
I expected it to use English as fallback language or handle the error in another way that wouldn't throw the errror.

## Minimal reproduction of the problem with instructions

Call the getBrowserLang() function in your app with all the languages removed from the browser's settings

## What is the motivation / use case for changing the behavior?

It crashes the app if there are no languages in the settings

## Environment
Firefox 75, 76, probably it's also reproduced in other versions of Firefox.
<pre><code>
Angular version: 9.1.9

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [x] Firefox version 75, 76
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

**Comment by itayod** [comment]
@mmrzk would you like to create a PR for it?

**Comment by Julien-Leos** [comment]
I have the same issue, `getBrowserLang()` returns `undefined` only server-side but returns the right value client-side.

I had to do this to avoid server-side crashing looking for `undefined.json` translation files.
```ts
const defaultLang = getBrowserLang() ? getBrowserLang() : 'en';
```

**Comment by shaharkazaz** [comment]
@Julien-Leos using this function is intended to be used in the client-side, the issue that @mmrzk has shown is the case where you don't have languages defined on your browser which is different from your case.
I will add that to the FAQ, but your solution is good, you can also write: `const defaultLang = getBrowserLang() || 'en';`

**Comment by shaharkazaz** [comment]
@mmrzk should be fixed in v2.17.2, can you verify? 😄
